index.html

```diff
@@ -4567,7 +4567,16 @@
     <section id="rtcrtpcodeccapability*">
       <h3>dictionary RTCRtpCodecCapability</h3>
       <p>The <dfn><code>RTCRtpCodecCapability</code></dfn> dictionary provides
-      information on the capabilities of a codec.</p>
+        information on the capabilities of a codec. Exactly one <code>RTCRtpCodecCapability</code>
+        will be present for each supported combination of parameters that must be identical
+        between the offer and answer. For example:</p>
```

Comment:
Since ORTC doesn't mandate using offers/answers, would it be accurate to say "identical between the RTCRtpSender and RTCRtpReceiver" instead? That makes sense to me.

At some point, should we also document which parameters fall under this "must be identical" category? For example, they're packetizationMode and profileLevelId for H.264:

> The parameters identifying a media format configuration for H.264 are profile-level-id and packetization-mode.  These media format configuration parameters (except for the level part of profile-level-id) MUST be used symmetrically; that is, the answerer MUST either maintain all configuration parameters or remove the media format (payload type) completely if one or more of the parameter values are not supported.

I didn't notice "except for the level part of profile-level-id" before. I guess that makes things slightly more complicated.